sunsynk/definitions.py

```diff
@@ -165,13 +165,26 @@
 _SENSORS.append(
     SelectRWSensor(243, "Priority Mode", options={0: "Battery first", 1: "Load first"})
 )
+PROG1_TIME = TimeRWSensor(250, "Prog1 Time")
+PROG2_TIME = TimeRWSensor(251, "Prog2 Time")
+PROG3_TIME = TimeRWSensor(252, "Prog3 Time")
+PROG4_TIME = TimeRWSensor(253, "Prog4 Time")
+PROG5_TIME = TimeRWSensor(254, "Prog5 Time")
+PROG6_TIME = TimeRWSensor(255, "Prog6 Time")
+PROG1_TIME.min_max(PROG6_TIME, PROG2_TIME)
+PROG2_TIME.min_max(PROG1_TIME, PROG3_TIME)
+PROG3_TIME.min_max(PROG2_TIME, PROG4_TIME)
+PROG4_TIME.min_max(PROG3_TIME, PROG5_TIME)
+PROG5_TIME.min_max(PROG4_TIME, PROG6_TIME)
+PROG6_TIME.min_max(PROG5_TIME, PROG1_TIME)
```

Comment:
I added a convenient method to be able to set the min and max sensors in one go. If there is a more elegant way to set up the dependencies taking into account that variables cannot be used before they are declared, then I'd happily implement any suggestion.